.github/workflows/robot/internal/review/review_test.go

```diff
@@ -37,26 +37,26 @@ func TestIsInternal(t *testing.T) {
 				c: &Config{
 					// Code.
 					CodeReviewers: map[string]Reviewer{
-						"1": Reviewer{Team: "Core", Owner: true},
-						"2": Reviewer{Team: "Core", Owner: true},
-						"3": Reviewer{Team: "Core", Owner: false},
-						"4": Reviewer{Team: "Core", Owner: false},
+						"1": Reviewer{Team: "Core", Owner: true, GithubUsername: "user1"},
+						"2": Reviewer{Team: "Core", Owner: true, GithubUsername: "user2"},
+						"3": Reviewer{Team: "Core", Owner: false, GithubUsername: "user3"},
+						"4": Reviewer{Team: "Core", Owner: false, GithubUsername: "user4"},
 					},
 					CodeReviewersOmit: map[string]bool{},
 					// Docs.
 					DocsReviewers: map[string]Reviewer{
-						"5": Reviewer{Team: "Core", Owner: true},
-						"6": Reviewer{Team: "Core", Owner: true},
+						"user5": Reviewer{Team: "Core", Owner: true},
+						"user6": Reviewer{Team: "Core", Owner: true},
```

Comment:
I think this is meant to be

```suggestion
						"5": Reviewer{Team: "Core", Owner: true, GithubUsername: "user5"},
						"6": Reviewer{Team: "Core", Owner: true, GithubUsername: "user6"},
```